source/linux/Foundational_Components/Kernel/Kernel_Drivers/Network/_K3-CPSW-common.rst

```diff
@@ -539,77 +573,87 @@ The main difference of the hard irqs deferral feature from ethtool interrupt pac
 
-The TI |__PART_FAMILY_NAME__| switch-config tool sources for |__PART_FAMILY_DEVICE_NAMES__|  SoC can be found at::
-
-       git@git.ti.com:switch-config/switch-config.git
-
-branch::
-
-       origin/am65x-v1.0
-
-Usage::
-
-       # switch-config -h
-       Switch configuration commands.....
-       switch-config -I,--ndev <dev> <command>
-
-       commands:
-       switch-config -d,--dump-ale :dump ALE table
-       switch-config -D,--dump=<0..9> :dump registers (0 - all)
-       switch-config -v,--version
-
-       dump values:
-        :1 - cpsw-nuss regs
-        :2 - cpsw-nuss-rgmii-status regs
-        :3 - cpsw-nuss-mdio regs
-        :4 - cpsw-nu regs
-        :5 - cpsw-nu-p0 regs
-        :6 - cpsw-nu-p1 regs
-        :7 - cpsw-nu-cpts regs
-        :8 - cpsw-nu-ale regs
-        :9 - cpsw-nu-ale-tbl regs
-
-Example, ALE table dump::
-
-        # switch-config -d
-        K3 cpsw dump version (1) len(6328)
-        ALE table dump ents(64):
-        0   : type: vlan , vid = 0, untag_force = 0x3, reg_mcast = 0x0, unreg_mcast = 0x0, member_list = 0x3
-        1   : type: ucast, addr = f4:84:4c:eb:a0:00, ucast_type = persistant, port_num = 0x0, Secure
-        2   : type: mcast, addr = ff:ff:ff:ff:ff:ff, mcast_state = f, no super, port_mask = 0x3
-        3   : type: mcast, addr = 01:00:5e:00:00:01, mcast_state = f, no super, port_mask = 0x3
-        4   : type: mcast, addr = 01:80:c2:00:00:00, mcast_state = f, no super, port_mask = 0x3
-        5   : type: mcast, addr = 01:80:c2:00:00:03, mcast_state = f, no super, port_mask = 0x3
-        6   : type: mcast, addr = 01:80:c2:00:00:0e, mcast_state = f, no super, port_mask = 0x3
-        8   : type: mcast, addr = 01:00:5e:00:00:fc, mcast_state = f, no super, port_mask = 0x3
-        9   : type: ucast, vid = 0, addr = 9c:b6:d0:89:0d:85, ucast_type = touched   , port_num = 0x1
-        26  : type: ucast, vid = 0, addr = c4:71:54:a9:6e:b4, ucast_type = touched   , port_num = 0x1
-        27  : type: ucast, vid = 0, addr = 00:25:22:a9:4c:b3, ucast_type = touched   , port_num = 0x1
-
-Example, CPTS registers dump::
-
-        switch-config -D7
-        K3 cpsw dump version (1) len(6328)
-        cpsw-nu-cpts regdump: num_regs(38)
-        0003d000:reg(4E8A2109)
-        0003d004:reg(00000C21)
-        0003d008:reg(00000000)
-        0003d00c:reg(00000000)
-        0003d010:reg(7EA3BA9B)
-        0003d014:reg(00000000)
-        0003d018:reg(00000000)
-        0003d01c:reg(00000000)
-        0003d020:reg(00000000)
-        0003d024:reg(00000000)
-        0003d028:reg(00000001)
-        0003d02c:reg(00000000)
-        0003d030:reg(00000000)
-        0003d034:reg(C7298A99)
-        0003d038:reg(03300000)
-        0003d03c:reg(00000000)
-        0003d040:reg(0000028E)
-        0003d044:reg(00000000)
-        0003d048:reg(00000000)
+The TI |__PART_FAMILY_NAME__| switch-config tool sources for |__PART_FAMILY_DEVICE_NAMES__|  SoC can be found at
+
+.. code-block:: console
```

Comment:
If the code block is not explicitly in reference to `console` output then it should be tagged `text` instead